scripts/build-locales.sh

```diff
@@ -16,31 +16,31 @@ cd "$(dirname "$0")"/../locale
 output_path="$1"
 
 # Check all the required tools are present
-if [ -z "$(which msgfmt)" ]; then
+if [ -z "$(command -v msgfmt)" ]; then
```

Comment:
This is a bash script, so you can use `type -P` which guarantees that it is an external command on $PATH.

`command -v` is POSIX sh and safe to use whenever you know "msgfmt" isn't a shell function -- `which` is a possibly not installed third-party package on most modern operating systems and a csh script that reports cshrc aliases on some (older) systems e.g. commercial unixes.

Why not switch to `[[` here as well?